src/Tizen.Applications.Common/Tizen.Applications.RPCPort/Parcel.cs

```diff
@@ -413,6 +472,20 @@ public short ReadShort()
             return b;
         }
 
+        /// <summary>
+        /// Reads an unsigned short value from the parcel object.
+        /// </summary>
+        /// <returns>The unsigned short data.</returns>
+        /// <since_tizen> 10 </since_tizen>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public ushort ReadUShort()
+        {
+            var bytes = Read(2);
```

Comment:
How about using sizeof operator to get size of the type?

Reply:
Ok, Thanks

Reply:
Thank you.